duckduckgo/iOS
# Update WireGuard version

Task/Issue URL: https://app.asana.com/0/414235014887631/1205173072817403/f
Tech Design URL:
CC:

**Description**:

This PR updates the version of WireGuard used by the app.



**Steps to test this PR**:
1. Check that the app builds and runs in Xcode 14, and that NetP works
2. Do **not** check that the app builds and runs in Xcode 15 - there is an unrelated compilation failure which is out of scope of this PR



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
